cmd/ak/cmd/up.go

```diff
@@ -33,6 +37,12 @@ var upCmd = common.StandardCommand(&cobra.Command{
 			return fmt.Errorf("fx app start: %w", err)
 		}
 
+		if readyFile != "" {
+			if err := os.WriteFile(readyFile, []byte("ready"), 0o644); err != nil {
+				fmt.Fprintf(os.Stderr, "write ready file: %v\n", err)
```

Comment:
Please don't use `os.Stdout` or `os.Stderr` directly in CLI commands. Right now it doesn't actually matter, but it may in the future, so we should minimize this. Instead, use `cmd.OutOrStdout` or `cmd.ErrOrStderr` - this allows Cobra to inject its own streams between/instead stdout and stderr.

Reply:
oh cool didn't know about this.